.circleci/config.yml

```diff
@@ -44,14 +44,15 @@ jobs:
     steps:
       - add_ssh_keys
       - checkout
-      - attach_workspace:
-          at: /tmp/workspace
       - run:
           name: Configure
           command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Build Assets
+          command: hokusai dev run 'yarn publish_assets'
```

Comment:
The `publish_assets` script depends on a `DEPLOY_ENV` variable that doesn't seem to be set any longer.

Reply:
`DEPLOY_ENV` is now listed in the `development.yml`, but I still think it needs to be explicitly passed here and [for production] below. If you look at the old commands being removed, you'll see what I mean.